contributor/contributors-guide.md

```diff
@@ -570,7 +570,65 @@ For details about governance policies for the Accumulo project view the followin
 - [Lazy Consensus][38]
 - [Voting][39]
 
-
+## Test a Accumulo release
+
+1. Set the release version, ID for staging repo, and alias to configure Maven with temporary settings:
+   ```shell
+   export RC_VERSION=1.9.0
+   export RC_STAGING=1070
+   alias mvn='mvn -s /tmp/accumulo-rc-maven.xml'
+   ```
+1. Create temporary Maven settings
+   ```shell
+   $ cat <<EOF >/tmp/accumulo-rc-maven.xml
+   <settings>
+     <profiles>
+       <profile>
+         <id>accumuloRC</id>
+         <repositories>
+           <repository>
+             <id>accumulorc</id>
+             <name>accumulorc</name>
+             <url>https://repository.apache.org/content/repositories/orgapacheaccumulo-\${env.RC_STAGING}/</url>
+           </repository>
+         </repositories>
+         <pluginRepositories>
+           <pluginRepository>
+             <id>accumulorcp</id>
+             <name>accumulorcp</name>
+             <url>https://repository.apache.org/content/repositories/orgapacheaccumulo-\${env.RC_STAGING}/</url>
+           </pluginRepository>
+         </pluginRepositories>
+       </profile>
+     </profiles>
+     <activeProfiles>
+       <activeProfile>accumuloRC</activeProfile>
+     </activeProfiles>
+   </settings>
+   EOF
+   ```
+1. If a new release candidate is made, update your staging repo ID.
+   ```shell
+   $ export RC_STAGING=1071
+   ```
+
+#### Run the integration tests of projects that use Accumulo
+
+1. Clone the [Accumulo Examples] project:
+    ```shell
+    $ git clone https://github.com/apache/accumulo-examples.git
+    ```
+1. Run the integration test
+    ```shell
+    $ mvn clean verify -Daccumulo.version=$RC_VERSION
```

Comment:
I'd rather inline the `-s /tmp/accumulo-rc-maven.xml` into these two examples. An alias could easily be forgotten, a user won't be able to use that same shell session for other maven tasks, without possible unexpected behavior.

Reply:
Fixed in c3749ada76db4